### PR TITLE
Centralised lodash

### PR DESCRIPTION
**Pull request by andre-dsm**
Fixed the lodash dependency and centralised it to services as per structure.

### CODE REVIEW

**Comment by haxxxton** [comment]
@andre-dsm thank you for this PR! Really appreciate you

File: src/services/utils/lodash.ts
[the remark this answers is not in the document]
Comment (andre-dsm) [reply src/services/utils/lodash.ts:1]: Yeah.. I debated over this too when created the folder name as lodash is so ubiquitous. I've split it as doing a find in replace in /utils seems ok vs everywhere.

File: package.json [hunk 1]
```diff
@@ -29,6 +29,7 @@
   "dependencies": {
     "@craco/craco": "^7.1.0",
     "clsx": "^1.2.1",
+    "lodash": "^4.17.21",
     "lodash.isequal": "^4.5.0",
```
[the remark this answers is not in the document]
Comment (andre-dsm) [reply package.json]: Done.

File: src/services/utils/lodash.ts [hunk 2]
```diff
@@ -0,0 +1,4 @@
+import { throttle, isEqual } from 'lodash';
```
[the remark this answers is not in the document]
Comment (andre-dsm) [reply src/services/utils/lodash.ts]: Done.

**Comment by haxxxton** [comment]
@andre-dsm awesome. Could you rebase this into 1-2 commits please. (I should really add a contributing guide that points to rebasing PR feedback rather than adding new commits) ready to merge other than that

**Comment by andre-dsm** [comment]
> @andre-dsm awesome. Could you rebase this into 1-2 commits please. (I should really add a contributing guide that points to rebasing PR feedback rather than adding new commits) ready to merge other than that

Done.

**Pull request merged**